### PR TITLE
ingest: Add `profiles/default` to make `-c1 --ri` unnecessary

### PR DESCRIPTION
**Pull request by corneliusroemer**

### CODE REVIEW

File: ingest/profiles/default/config.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+cores: all
+rerun-incomplete: true
+printshellcmds: true
+reason: true
```
Comment (victorlin) [inline ingest/profiles/default/config.yaml:4]: These options are newly introduced, but seems reasonable to me.

**Review by victorlin — APPROVED** [review]

File: ingest/profiles/default/config.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+cores: all
```
Comment (victorlin) [inline ingest/profiles/default/config.yaml:1]: Changed from `1` to `all` but I think that's fine.

Comment (corneliusroemer) [reply ingest/profiles/default/config.yaml:1]: I don't know why we ever used 1 in the first place

File: ingest/profiles/default/config.yaml (hunk 1 above)
Comment (corneliusroemer) [reply ingest/profiles/default/config.yaml:4]: Yep, rerun incomplete just avoids an error, it's something almost everyone adds all the time anyways when running builds

**Pull request merged**